code/AssetLib/Blender/BlenderScene.h

```diff
@@ -884,7 +876,7 @@ struct Tex : ElemBase {
     //char use_nodes;
 
     Tex() :
-            ElemBase(), imaflag(ImageFlags_INTERPOL), type(Type_CLOUDS), ima() {
+            imaflag(ImageFlags_INTERPOL), type(Type_CLOUDS) {
```

Comment:
base class default c’tors are always called; `std::shared_ptr` is always default-initialized